repo/packages/S/spark/12/marathon.json.mustache

```diff
@@ -23,6 +30,15 @@
         "SPARK_SSL_PROTOCOL": "{{security.ssl.protocol}}",
         "SPARK_SSL_ENABLEDALGORITHMS": "{{security.ssl.enabledAlgorithms}}",
 {{/security.ssl.enabled}}
+{{#security.authn.enabled}}
+        "DCOS_SERVICE_ACCOUNT_CREDENTIAL": { "secret": "serviceCredential" },
+        "MESOS_MODULES": "file:///opt/mesosphere/etc/mesos-scheduler-modules/dcos_authenticatee_module.json",
+        "MESOS_AUTHENTICATEE": "com_mesosphere_dcos_ClassicRPCAuthenticatee",
+        "SSL_CERT_FILE": "{{security.authn.ssl_cert_file}}",
+        "SSL_KEY_FILE": "{{security.authn.ssl_key_file}}",
+        "SSL_CA_DIR": "{{security.authn.ssl_ca_dir}}",
+        "SSL_ENABLED": "true",
```

Comment:
@mgummelt split each of these SSL_'s out into their own conditional sections, guarded by conditions like `{{#security.mesos.ssl.cert_file}}`